core/src/main/scala/zio/logging/Logger.scala

```diff
@@ -0,0 +1,64 @@
+package zio.logging
+
+import zio.clock._
+import zio.console._
+import zio.internal.Tracing
+import zio.internal.stacktracer.Tracer
+import zio.internal.stacktracer.ZTraceElement.{ NoLocation, SourceLocation }
+import zio.internal.stacktracer.impl.AkkaLineNumbersTracer
+import zio.internal.tracing.TracingConfig
+import zio.{ FiberRef, UIO, ZIO }
+
+/**
+ * A logger of strings.
+ */
+trait Logger extends LoggerLike[String]
+object Logger {
+
+  private val tracing = Tracing(Tracer.globallyCached(new AkkaLineNumbersTracer), TracingConfig.enabled)
+
+  private[logging] def classNameForLambda(lambda: => AnyRef) =
+    tracing.tracer.traceLocation(() => lambda) match {
+      case SourceLocation(_, clazz, _, _) => Some(clazz)
+      case NoLocation(_)                  => None
+    }
+
+  /**
+   * Constructs a logger provided the specified sink.
+   */
+  def make(logLevel: LogLevel, logger: (LogContext, => String) => UIO[Unit]): UIO[Logger] =
```

Comment:
This is like a "default log level". I'm not sure if that's good or not. But anyway, you can generalize it by changing it to `LogContext`, and then the user can supply defaults for everything, if they really want. Maybe default it to `LogContext.empty`

Reply:
I don't like that because it's confusing. I added this because when you start default level was ```Off``` I didn't see logs I thought something wrong with my setup. I thing users might end up in the same situation. I changed neutral value in ```LogAnnotation.Level``` to be ```Info``` then user will see logs and everyone will be happy.